From 1f6ad0e83e68c659ea226178d36b83e8d1754372 Mon Sep 17 00:00:00 2001
From: Leland Carey <lc@gracefulrestart.com>
Date: Tue, 1 Sep 2020 12:42:45 -0700
Subject: [PATCH] fixing waashero-config.php path

---
 waas-hero-must-use.php | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/waas-hero-must-use.php b/waas-hero-must-use.php
index c8b2bb8..6469f98 100644
--- a/waas-hero-must-use.php
+++ b/waas-hero-must-use.php
@@ -50,11 +50,11 @@ defined( 'ABSPATH' ) or die( 'Cheatin&#8217; uh?' );
  */
 
 // require_once WPMU_PLUGIN_DIR.'/wp-stateless/wp-config-transformer/src/WPConfigTransformer.php';
-$dir = trim( ABSPATH, "/" );
-if( !file_exists( $dir. "/waashero-config.php" ) ) {
-    $file = fopen( $dir. "/waashero-config.php", 'a' );
+$dir = trim( ABSPATH );
+if( !file_exists( WPMU_PLUGIN_DIR. "waashero-config.php" ) ) {
+    $file = fopen( WPMU_PLUGIN_DIR. "waashero-config.php", 'a' );
     fclose( $file );
-    file_put_contents( $dir. "/waashero-config.php", file_get_contents( $dir. "/wp-config.php" ) );
+    file_put_contents( WPMU_PLUGIN_DIR. "waashero-config.php", file_get_contents( $dir. "wp-config.php" ) );
 }
 require_once WPMU_PLUGIN_DIR.'/waashero/wp-stateless/wp-stateless-media.php';
-require WPMU_PLUGIN_DIR.'/waashero/waashero.php';
\ No newline at end of file
+require WPMU_PLUGIN_DIR.'/waashero/waashero.php';
-- 
1.8.3.1

